internal/crypto/src/tests/raw_signature/rust_native/validators.rs

```diff
@@ -52,7 +49,7 @@ fn es256_bad_signature() {
 }
 
 #[test]
-#[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
```

Comment:
We need the `cfg_attr` to remain present so that we keep these test running for WASM.

Sadly, the `#[test]` attribute is ignored by the WASM test runner.

I'll add them back. Marking changes requested, but I'll make the changes.